### ISSUE TITLE
How to disable "FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`."

### ISSUE THREAD

**Issue by pengyu** [h5py/h5py#995]
I see the following warning when I use h5py. Is there a way to disable it? Thanks.

```
/home/py/.linuxbrew/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
```

**Comment by rmvanhees** [comment]
This is a known issue. Wait for release 2.7.2 or use the warnings-module to
ignore this warning.

import warnings
with warnings.catch_warnings():
    warnings.filterwarnings("ignore",category=FutureWarning)
    import h5py

Greetings, Richard

**Comment by tacaswell** [comment]
Fixed by #985
